pennylane/transforms/qfunc_transforms.py

```diff
@@ -78,12 +79,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class NonQueuingTape(qml.queuing.AnnotatedQueue):

```

Comment:
How come this was changed?

Reply:
Yes we can revert this after our discussion regarding imports, I originally changed this because we import most things this way and my local linter (not pylint) wasn't recognizing the class.